interface/forms/button_ros/new.php

```diff
@@ -0,0 +1,667 @@
+		
+		<span class="checkbox-button">
+			<input id="ENT9" name="ENT9" type="checkbox" value="on" >
+			<label for="ENT9"> <?php echo xlt('Mouth Dryness');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENT10" name="ENT10" type="checkbox" value="on" >
+			<label for="ENT10"> <?php echo xlt('Problem Swallowing');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="ENT11" name="ENT11" type="checkbox" value="on" >
+			<label for="ENT11"> <?php echo xlt('Dental Problems');?></label>
+		</span>
+		</TD>	
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="ENT_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>	
+	
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('RESP:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="RESP1" name="RESP1" type="checkbox" value="on" >
+			<label for="RESP1"> <?php echo xlt('Speech Problem');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="RESP2" name="RESP2" type="checkbox" value="on" >
+			<label for="RESP2"> <?php echo xlt('Shortness of Breath');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="RESP3" name="RESP3" type="checkbox" value="on" >
+			<label for="RESP3"> <?php echo xlt('Orthopnea');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="RESP4" name="RESP4" type="checkbox" value="on" >
+			<label for="RESP4"> <?php echo xlt('Cough');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="RESP5" name="RESP5" type="checkbox" value="on" >
+			<label for="RESP5"> <?php echo xlt('Cough w/Sputum');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="RESP6" name="RESP6" type="checkbox" value="on" >
+			<label for="RESP6"> <?php echo xlt('Hemoptysis');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="RESP7" name="RESP7" type="checkbox" value="on" >
+			<label for="RESP7"> <?php echo xlt('Wheezing');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="RESP_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>
+
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('CVS:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="CVS1" name="CVS1" type="checkbox" value="on" >
+			<label for="CVS1"> <?php echo xlt('Chest Pain');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="CVS2" name="CVS2" type="checkbox" value="on" >
+			<label for="CVS2"> <?php echo xlt('Palpitations');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="CVS3" name="CVS3" type="checkbox" value="on" >
+			<label for="CVS3"> <?php echo xlt('Leg Pain w/Exercise');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
```

Comment:
I always got shot down for punctuation in the information to be translated it breaks the translation